tasks/install.yml

```diff
@@ -1,44 +1,48 @@
 ---
 
-- name: Install essential packages
+- name: Install Essential Packages
   become: yes
   apt:
     name: "{{ item }}"
+    update_cache: yes
```

Comment:
I think cache updates should be moved to a task at the playbook level rather than in individual tasks like this.